src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java

```diff
@@ -643,6 +643,77 @@ private Mono<Void> assertLoadCreationInfosNotEmpty(LoadCreationInfos loadCreatio
         return loadCreationInfos == null ? Mono.error(new NetworkModificationException(CREATE_LOAD_ERROR, "Missing required attributes to create the load")) : Mono.empty();
     }
 
+    private Load modifyLoad(Network network, LoadModificationInfos loadModificationInfos) {
+        // modify the load
+        Load load = network.getLoad(loadModificationInfos.getEquipmentId());
+        if (load == null) {
+            throw new NetworkModificationException(LOAD_NOT_FOUND, "Load " + loadModificationInfos.getEquipmentId() + " does not exist in network");
+        }
+
+        if (loadModificationInfos.getLoadType() != null) {
+            load.setLoadType(loadModificationInfos.getLoadType().applyModification(load.getLoadType()));
+        }
+        if (loadModificationInfos.getActivePower() != null) {
+            load.setP0(loadModificationInfos.getActivePower().applyModification(load.getP0()));
+        }
+        if (loadModificationInfos.getReactivePower() != null) {
+            load.setQ0(loadModificationInfos.getReactivePower().applyModification(load.getQ0()));
+        }
+        // TODO connectivity modification
+        return load;
+    }
+
+    private List<EquipmentModificationInfos> execModifyLoad(NetworkStoreListener listener,
+                                                            LoadModificationInfos loadModificationInfos,
+                                                            ReporterModel reporter,
+                                                            Reporter subReporter) {
+        Network network = listener.getNetwork();
+        UUID networkUuid = listener.getNetworkUuid();
+
+        return doAction(listener, () -> {
+            if (listener.isApplyModifications()) {
+                try {
+                    // modify the load in the network
+                    modifyLoad(network, loadModificationInfos);
+
+                    subReporter.report(Report.builder()
+                            .withKey("loadModification")
+                            .withDefaultMessage("Load with id=${id} modified")
+                            .withValue("id", loadModificationInfos.getEquipmentId())
+                            .withSeverity(new TypedValue("LOAD_MODIFICATION_INFO", TypedValue.INFO_LOGLEVEL))
+                            .build());
+                } catch (NetworkModificationException exc) {
+                    subReporter.report(Report.builder()
+                            .withKey("loadModification")
+                            .withDefaultMessage(exc.getMessage())
+                            .withValue("id", loadModificationInfos.getEquipmentId())
+                            .withSeverity(new TypedValue("LOAD_MODIFICATION_ERROR", TypedValue.ERROR_LOGLEVEL))
+                            .build());
+                }
+            }
+
+            // add the load creation entity to the listener
```

Comment:
Typo : add the load modification entity to the listener

Reply:
Corrected